pkg/apis/core/v1beta1/kubefedcluster_types.go

```diff
@@ -23,6 +23,14 @@ import (
 	"sigs.k8s.io/kubefed/pkg/apis/core/common"
 )
 
+type TLSValidation string
+
+const (
```

Comment:
Could we add a meaningful prefix to these constants e.g. `TLSAll`, `TLSCheckAll`, etc?

Reply:
I've updated the names of TLSValidations to start with 'TLS'.